src/style.css

```diff
@@ -1,3 +1,67 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
```

Comment:
better to move styles for font to html selector.
here good place to reset default margin for body

Reply:
still not fixed